src/components/Header.tsx

```
@@ -1,56 +1,79 @@
+              >
+                Join Waitlist
+              </button>
+            </div>
+
+            {/* Mobile Menu Button */}
+            <button
+              className='md:hidden p-2 rounded-lg hover:bg-gray-100 transition-colors'
+              onClick={() => setIsMobileMenuOpen(true)}
+              aria-label='Open menu'
+              aria-expanded={isMobileMenuOpen}
             >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)